docs/docs/guides/clean-workflows.md

```diff
@@ -0,0 +1,76 @@
+# Clean Workflows
+
+Director allows you to permanently delete useless records of your workflows based on a retention value that is a positive integer.
+For example, you can use it to keep only the 10 last records of a workflow every day at midnight.
+
+This feature leverages [celery beat](https://docs.celeryproject.org/en/stable/userguide/periodic-tasks.html#crontab-schedules).
+
+# Prerequisites
+
+This feature requires that you start a Celery Beat scheduler.
+
+You can find more details about the periodic workflows commands in [the build guide](https://ovh.github.io/celery-director/guides/build-workflows/).
+
+
+
+Let's imagine the following workflows :
+
+```yaml
+---
+# Simple ETL example
+#
+# +-----------+      +-------------+      +--------+
+# |  EXTRACT  +----->+  TRANSFORM  +----->+  LOAD  |
+# +-----------+      +-------------+      +--------+
+#
+example.ETL:
+  tasks:
+    - EXTRACT
+    - TRANSFORM
+    - LOAD
+  retention: 2
+
+
+# Group of tasks example
+#
+# +----------+       +----------+
+# |  RANDOM  |       |  RANDOM  |
+# +----+-----+       +-----+----+
+#      |     +-------+     |
+#      +---->+  ADD  <-----+
+#            +-------+
+#
+example.RANDOMS:
+  tasks:
+    - GROUP_RANDOMS:
+        type: group
+        tasks:
+          - RANDOM
+          - RANDOM
+    - ADD
+
+```
+
+
+# How it works
+
+For each workflow that is present in your *workflows.yml*, the cleaning process will:
```

Comment:
Please don't explain how your code works but how the user can use this feature ^^